src/Dfc.CourseDirectory.Core/Content.resx

```diff
@@ -147,6 +147,21 @@
   <data name="ERROR_COURSERUN_COURSE_WEB_PAGE_MAXLENGTH" xml:space="preserve">
     <value>&lt;TODO&gt;</value>
   </data>
+  <data name="ERROR_COURSE_HOW_YOU_WILL_BE_ASSESSED_MAXLENGTH" xml:space="preserve">
```

Comment:
Would be good to back out the changes in this file (they seem to only be reordering but it's making the diff a little messy).